conformance/fhir-ig-davinci-pdex-formulary/CHANGELOG.md

```diff
@@ -25,6 +25,25 @@ Source - http://hl7.org/fhir/us/davinci-drug-formulary/STU1.1 retrieved on May 2
 3. if its a new version
    - ensure references from the previous package are version-specific (e.g. to avoid a 1.0.1 profile from picking up a 1.1.0 valueset during validation)
    - add a new provider (`src/main/java` and `src/main/resources/META-INF`)
-   - create tests for this new version (update ConstraintGeneratorTest and PlanNetResourceProviderTest)
+   - create tests for this new version (update ConstraintGeneratorTest and FormularyResourceProviderTest)
+4. execute ResourceProcessor from src/test/java with the proper arguments
+5. execute SearchParameterAugmenter from src/test/java with the proper arguments
+
+## PDEX US Drug Formulary 2.0.0
```

Comment:
my intent was to have these `h2` elements (`##`) all above the "Steps to update".
so please remove the "Steps to update" section that is above here and keep just the one after.  the resulting structure should be:
* Changes
   * 1.0.1
   * 1.1.0
   * 2.0.0
* Steps to update

Reply:
This has been fixed. moved the changes made to 2.0.0 above steps to update and removed the redundant one.